### ISSUE TITLE
Words list not updates

### ISSUE THREAD

**Issue by sank8dalvi** [skribbliohints/skribbliohints.github.io#3]
I want to contribute some words which I encounter but they are not there in words.json.
What's the process to add some words there?

**Comment by skribbliohints** [comment]
If you can provide some words that words.json is missing, please do.  Leave the words in a comment on this issue.

**Comment by sank8dalvi** [comment]
1. grim reaper

**Comment by skribbliohints** [comment]
For some reason the bot has not found the word "grim reaper".  The bot has found all words available to it.  So this word is not available to the bot for some reason.  I have been running the bot in private games in English over the last few days and it still has not found this word.  Also, the data indicates that the bot gets the same words in public games as private games.

All data in words.json was generated by the bot.  If words.json is changed manually, we lose some guarantees about the data.  So "grim reaper" cannot be added easily.

So I do not plan to add "grim reaper".  But if I get enough words that the bot is missing, I would consider adding them somehow.  I am leaving this issue open for now to invite people to comment with words the bot is missing.

**Comment by Samuel-KW** [comment]
As of 9/14/20 the wordlist is still up to date, I found a quick way to obtain all the words from any language and it seems as if nothing has changed. If you want the method of quickly obtaining all the words (hours not months), let me know.

If you want to run a quick check and compare the words, I included all the words in a pastebin. There were 2,298 different words that were found.

[Link to pastebin with wordlist](https://pastebin.com/raw/rEaCYk77)
